src/components/Launch/LaunchWorkflowForm/LaunchWorkflowForm.tsx

```diff
@@ -53,11 +58,38 @@ function getComponentForInput(input: InputProps) {
     }
 }
 
+function generateFetchSearchResults(
+    { listWorkflows }: APIContextValue,
+    workflowId: WorkflowId
+) {
+    return async (query: string) => {
+        const { entities: workflows } = await listWorkflows(workflowId, {
+            filter: [
+                {
+                    key: 'version',
+                    operation: FilterOperationName.CONTAINS,
+                    value: query
+                }
+            ]
+        });
+        const options = workflowsToWorkflowSelectorOptions(workflows);
+        if (options.length > 0) {
+            options[0].description = 'latest';
```

Comment:
Is this something that is uniquely assigned here? Like, elsewhere that `workflowsToWorkflowSelectorOptions` is used, would you expect a nonzero length where the zeroth element is _not_ `latest`? Just checking to see if this is logic that should be contained in the function itself.

Reply:
This is specific to the form. I didn't want to hard-code that logic into the selector options function.